packages/framework/src/Model/Customer/Exception/CustomerUserNotFoundByEmailAndDomainException.php

```diff
@@ -4,7 +4,7 @@
 
 use Exception;
 
-class UserNotFoundByEmailAndDomainException extends UserNotFoundException
+class CustomerUserNotFoundByEmailAndDomainException extends CustomerUserNotFoundException
```

Comment:
I think, that this changes should be committed in previous commit "renamed User* objects to CustomerUser*"

Reply:
It should, but I will leave it as is.